orc8r/cloud/deploy/terraform/orc8r-helm-aws/main.tf

```diff
@@ -34,8 +34,155 @@ resource "helm_release" "orc8r" {
   version             = var.orc8r_chart_version
   keyring             = ""
   timeout             = 600
+  values              = [data.template_file.orc8r_values.rendered]
 
-  values = [templatefile("${path.module}/templates/orc8r-values.tpl", {
+  set_sensitive {
+    name  = "controller.spec.database.pass"
+    value = var.orc8r_db_pass
+  }
+
+  set_sensitive {
+    name  = "nms.magmalte.env.mysql_pass"
+    value = var.nms_db_pass
+  }
+}
+
+resource "helm_release" "lte-orc8r" {
+  count = (
+    var.orc8r_deployment_type == "fwa" ||
+    var.orc8r_deployment_type == "federated_fwa" ||
+    var.orc8r_deployment_type == "all"
+  ) ? 1 : 0
+
+  name                = "lte-orc8r"
+  namespace           = kubernetes_namespace.orc8r.metadata[0].name
+  repository          = var.helm_repo
+  repository_username = var.helm_user
+  repository_password = var.helm_pass
+  chart               = "lte-orc8r"
+  version             = var.lte_orc8r_chart_version
+  keyring             = ""
+  timeout             = 600
+  values              = [data.template_file.orc8r_values.rendered]
+
+  set_sensitive {
+    name  = "controller.spec.database.pass"
+    value = var.orc8r_db_pass
+  }
+
+  set_sensitive {
+    name  = "nms.magmalte.env.mysql_pass"
+    value = var.nms_db_pass
+  }
+}
+
+resource "helm_release" "feg-orc8r" {
+  count = (
+    var.orc8r_deployment_type == "federated_fwa" ||
+    var.orc8r_deployment_type == "all"
+  ) ? 1 : 0
+
+  name                = "feg-orc8r"
+  namespace           = kubernetes_namespace.orc8r.metadata[0].name
+  repository          = var.helm_repo
+  repository_username = var.helm_user
+  repository_password = var.helm_pass
+  chart               = "feg-orc8r"
+  version             = var.feg_orc8r_chart_version
+  keyring             = ""
+  timeout             = 600
+  values              = [data.template_file.orc8r_values.rendered]
+
+  set_sensitive {
+    name  = "controller.spec.database.pass"
+    value = var.orc8r_db_pass
+  }
+
+  set_sensitive {
+    name  = "nms.magmalte.env.mysql_pass"
+    value = var.nms_db_pass
+  }
+}
+
+resource "helm_release" "cwf-orc8r" {
+  count               = var.orc8r_deployment_type == "all" ? 1 : 0
+  name                = "cwf-orc8r"
+  namespace           = kubernetes_namespace.orc8r.metadata[0].name
+  repository          = var.helm_repo
+  repository_username = var.helm_user
+  repository_password = var.helm_pass
+  chart               = "cwf-orc8r"
+  version             = var.cwf_orc8r_chart_version
+  keyring             = ""
+  timeout             = 600
+  values              = [data.template_file.orc8r_values.rendered]
+
+  set_sensitive {
+    name  = "controller.spec.database.pass"
+    value = var.orc8r_db_pass
+  }
+
+  set_sensitive {
+    name  = "nms.magmalte.env.mysql_pass"
+    value = var.nms_db_pass
+  }
+}
+
+
+resource "helm_release" "fbinternal-orc8r" {
```

Comment:
should we perhaps cleanup fbinternal and wifi pods going forward ?

Reply:
These modules still provide services that we need  - though I think the wifi one may be able to be removed soon.